core/src/main/java/io/kestra/core/models/tasks/retrys/Random.java

```diff
@@ -33,4 +35,11 @@ public <T> RetryPolicy<T> toPolicy() {
         return policy
             .withDelay(minInterval.toMillis(), maxInterval.toMillis(), ChronoUnit.MILLIS);
     }
+
+    @Override
+    public Instant getNextDate(Integer attemptCount, Instant lastAttempt) {
+        SecureRandom random = new SecureRandom();
```

Comment:
```suggestion
        Random random = new Random();
```

SecureRandom is for cryptographic code (why the 'secure') and very costly.

Reply:
If I dont use SecureRandom(), Sonarlint fail my CI :cry: 
Do I discard is warning and put random back so ?